images/dashboard/dashboard.py

```diff
@@ -111,6 +126,13 @@ def plot_interactions(archive_select):
     plot_daily_activity,
 )
 
+plot_wordtrends_boxed = pn.Column(
+    pn.pane.Markdown(
+        "#### it computes and plot word counts over time, on aggregated mailing lists' data. it exports emails that contains selected words"
```

Comment:
How about: "This plot show the occurrence of selected words in the mailing list over time."